README.md

```diff
@@ -67,28 +38,6 @@ You can then run the [GitHub Action](https://github.com/alphagov/seal/actions) s
 
 If you don't want to post to Slack you can add a `DRY: true` env to your workflow and the output will only show in the logs.
 
-### Slack configuration
```

Comment:
I'd keep this section (especially as https://github.com/alphagov/seal/tree/main/images/emoji still exists). If our Slack instance is ever reset, for example, it would be good to know what needs setting.
Could move it into its own file in `docs/` if you like?